tests/rptest/services/redpanda.py

```diff
@@ -170,10 +211,19 @@ def __init__(self,
         self._enable_sr = enable_sr
         self._log_level = self._context.globals.get(self.LOG_LEVEL_KEY,
                                                     self.DEFAULT_LOG_LEVEL)
-        self._num_cores = num_cores
         self._admin = Admin(self)
         self._started = []
         self._security_config = dict()
+        self._admin = Admin(self)
+        self._started = []
+        self._security_config = dict()
+
+        if resource_settings is None:
+            resource_settings = ResourceSettings()
+        self._resource_settings = resource_settings
+
+        # client is intiialized after service starts
+        self._client = None
```

Comment:
I think that `self._client` is now (as of a few days ago) an unused instance variable